doc/releases/releases.yml

```diff
@@ -241,8 +246,10 @@ releases:
 
 development:
   releases:
-    - version: 15.1.0
-      released: 2020-01-29
+#    - version: 15.1.0
+#      released: 2020-01-29
+#    - version: 15.1.1
+#      released: 2020-03-13
```

Comment:
instead of commenting these lines out, you could add `skip_ref: true` - then they would behave like the previous development versions

Reply:
also, `15.1.1` should come first, with `15.1.0` following (the list is in reverse chronological, and numerical, order)